README.md

```diff
@@ -147,6 +147,26 @@ Once that's done, you can run the manager and API services:
     $ sudo esi-leap-api
 ```
 
+### Installation using Containerization
+
+By encapsulating the ESI Leap into a container, all the necessary dependencies, configurations and services are bundled into a single package.
+
+Please make sure to follow the instructions in the [Configuration](#configuration) section to generate the esi-leap.conf file, and ensure that it is present in the directory from which you are running the `podman build` command.
```

Comment:
Since we're no longer copying the configuration file into the container image during the build process itself, it is no longer necessary for the file to be in the working directory.